app/src/main/java/fr/neamar/kiss/ui/KeyboardScrollHider.java

```diff
@@ -162,11 +163,19 @@ public boolean onTouch(View v, MotionEvent event) {
                 this.lastMotionEvent = null;
 
                 if (!this.resizeDone) {
+                    // Hide the keyboard if the user has scrolled down by about half a result item
+                    if (isScrolled()) {
+                        if (this.handler.isAllowHideKeyboardEnabled()) {
+                            this.handler.hideKeyboard();
+                        }
+                        this.handler.applyScrollSystemUi();
+                    }
                     ValueAnimator animator = ValueAnimator.ofInt(
                             this.list.getHeight(),
                             this.listParent.getHeight()
                     );
-                    animator.setDuration(250);
+                    // Duration may need increased for better animations but 250 is too high
+                    animator.setDuration(0);
```

Comment:
Without this additional animation it feels a lot better then before.
Can you remove it completely instead if just disabling it with duration 0?